model/project.go

```diff
@@ -546,6 +546,9 @@ type PluginCommandConf struct {
 	// Vars defines variables that can be used within commands.
 	Vars map[string]string `yaml:"vars,omitempty" bson:"vars,omitempty"`
 
+	// RetryOnFailure indicates whether the task should be retried if this command fails.
+	RetryOnFailure bool `yaml:"retry_on_failure,omitempty" bson:"retry_on_failure,omitempty"`
```

Comment:
[non-blocking]
It looks like this was changed from `auto_retry` to `retry_on_failure` since the PR was created. Can you please update the PR description when you have a chance? 
Thank you!

Reply:
Done